doc/user/layouts/shortcodes/self-managed/versions/curl-sample-files-local-install.html

```diff
@@ -1,5 +1,5 @@
 ```shell
-mz_version=v0.130.4
+mz_version=v0.130.7
```

Comment:
I think  for the operator/values.yaml ... we might need to curl a different tag version?